DESCRIPTION

```diff
@@ -12,18 +12,18 @@ Encoding: UTF-8
 LazyData: true
 Remotes:  
     mrc-ide/individual@0.0.8,
-    mrc-ide/squire@v0.5.5
+    mrc-ide/squire@v0.5.7
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.1
 Suggests:
     testthat,
     mockery
 Imports:
     individual,
-    squire (>= 0.5.5),
+    squire,
```

Comment:
can you add the version requirement here back please? Presumably 0.5.7 as you updated the version installed